tests/common/test_cgroupapi.py

```diff
@@ -119,59 +122,31 @@ def controller_operation(controller):
             self.assertEqual(len(successful_controllers), 1, 'The operation was not executed on unexpected controllers: {0}'.format(successful_controllers))
 
             args, kwargs = mock_logger_warn.call_args
-            message = args[0]
-            self.assertIn('Error in cgroup controller "cpu": A test exception.', message)
-
-
-class FileSystemCgroupsApiTestCase(AgentTestCase):
-
-    def setUp(self):
-        AgentTestCase.setUp(self)
-
-        self.cgroups_file_system_root = os.path.join(self.tmp_dir, "cgroup")
-        os.mkdir(self.cgroups_file_system_root)
-        os.mkdir(os.path.join(self.cgroups_file_system_root, "cpu"))
-        os.mkdir(os.path.join(self.cgroups_file_system_root, "memory"))
+            (message_format, controller, error, message) = args
+            self.assertEquals(message_format, 'Error in cgroup controller "{0}": {1}. {2}')
```

Comment:
So this will abstract away whatever's in `{}`? Cool, did not know this.

Reply:
This was actually needed because I changed the statement that logs this message. Previously it was logging a string formatted with string.format, now it lets the logger do the formatting so the first argument is actually the formatting string.